### PR TITLE
Emailrepio

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21967

## Description
Added EmailRep integration

## Minimum version of Demisto
- [V] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [V ] No

## Must have
- [V] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Emailrep/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "certification": "verified",
+    "categories": [],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "dependencies": {
+        "CommonScripts": {
+            "mandatory": true,
+            "display_name": "Common Scripts"
+        }
```
Comment (ronykoz) [inline Packs/Emailrep/pack_metadata.json:19]: remove this section, it's autogenerated in the build

File: Packs/Emailrep/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "EmailRep",
+    "description": "Provide email address reputation and reports from EmailRep.io.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://emailrep.io",
+    "email": "",
+    "created": "2020-03-10T08:37:18Z",
```
Comment (ronykoz) [inline Packs/Emailrep/pack_metadata.json:9]: support should be XSOAR
take a look at Packs/AutoFocus/pack_metadata.json for instance

File: Packs/Emailrep/TestPlaybooks/TestEmailRepPlaybook.yml [hunk 3]
```diff
@@ -0,0 +1,569 @@
+id: TestEmailRepPlaybook
+version: 9
```
Comment (ronykoz) [inline Packs/Emailrep/TestPlaybooks/TestEmailRepPlaybook.yml:2]:
```suggestion
version: -1
```

File: Packs/Emailrep/Integrations/EmailRep/EmailRep.py [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        if description:
+            request_params["description"] = description
+
+        if timestamp:
+            request_params["timestamp"] = timestamp
+
+        if expires:
+            request_params["expires"] = expires
+
+        demisto.debug(f"DDDD:  {request_params}")
```
Comment (ronykoz) [inline Packs/Emailrep/Integrations/EmailRep/EmailRep.py:52]: remove/format please

**Review by ronykoz — COMMENTED** [review]
Looks good!

Is the image taken from the UI team?

File: Packs/Emailrep/Integrations/EmailRep/EmailRep.py [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+        email_score_list.append(email_context)
+
+    readable_output = tableToMarkdown('Email List', email_data_list)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_NAME}.EmailScore',
+        outputs_key_field='email',
+        outputs=email_data_list,
+        indicators=email_score_list
```
Comment (DeanArbel) [inline Packs/Emailrep/Integrations/EmailRep/EmailRep.py:135]: Please split this to multiple `CommandResults` (1 per email) with a single indicator/Human Readable/output per result.
The handling of multiple indicators is due to change as detailed in https://github.com/demisto/etc/issues/29016.
Do note, `return_results` currently does not support list of `CommandResults` - so for now you'll need to handle it via the integration, like the changes that are introduced here: https://github.com/demisto/content/blob/84f6226db4034d85dbfecc60e545d2254aea6ea9/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L2771

**Comment by DinaMeylakh** [comment]
> Is the image taken from the UI team?

No, it's the one from their API site. Who should I talk to?

File: Packs/Emailrep/Integrations/EmailRep/EmailRep.py (hunk 5 above)
Comment (DinaMeylakh) [reply Packs/Emailrep/Integrations/EmailRep/EmailRep.py]: No problem, Arseny wanted this done for one email in each command anyway.

**Review by ronykoz — APPROVED** [review]
Let's update the image and do the demo and we are good to go

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

File: Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+    )
+
+    readable_output = tableToMarkdown('Email', email_data)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_NAME}.Email',
+        outputs_key_field='id',
+        outputs=email_data,
+        indicators=[account_context]
```
Comment (DeanArbel) [inline Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py:126]:
```suggestion
        indicator=account_context
```

Comment (DinaMeylakh) [reply Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py:126]: Is this change already merged? My tests seem to fail

Comment (ronykoz) [reply Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py:126]: yeah i think so

**Pull request merged**